packages/lism-ui/src/components/Popover/_style.css

```
@@ -0,0 +1,137 @@
+@layer lism-block {
+  .b--popover {
+    display: inline-block;
+    anchor-scope: --popover;
+  }
+  .b--popover_trigger {
+    anchor-name: --popover;
+  }
+
+  .b--popover_popup {
+    --duration: var(--popover-duration, 0.15s);
+    border: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)